lds_merge_property_data.pyt

```diff
@@ -398,6 +398,9 @@ def create_relationships(options):
         add_title_relationships(options.output_file_gdb)
     return True
 
+"""

```

Comment:
@iharrison can we please re-write the code so the tool can be run from the CLI without having to uncomment code? That's pretty horrible.